zcash_client_sqlite/src/wallet/init/migrations/full_account_ids.rs

```diff
@@ -53,7 +56,9 @@ impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
             seed_fingerprint: SeedFingerprint::from_bytes([0; 32]),
             account_index: zip32::AccountId::ZERO,
         });
-        let account_kind_imported = account_kind_code(AccountSource::Imported);
+        let account_kind_imported = account_kind_code(AccountSource::Imported {
+            purpose: AccountPurpose::ViewOnly,
```

Comment:
Add a comment that the value of this `purpose` field is irrelevant to `account_kind_code`, and chosen arbitrarily.